osu.Game.Rulesets.Catch/Objects/JuiceStream.cs

```diff
@@ -110,7 +110,11 @@ protected override void CreateNestedHitObjects()
             }
         }
 
-        public double EndTime => StartTime + this.SpanCount() * Path.Distance / Velocity;
+        public double EndTime
+        {
+            get => StartTime + this.SpanCount() * Path.Distance / Velocity;
+            set => throw new System.NotImplementedException();
```

Comment:
If I'm understanding this right, using this setter is directly incorrect - so I'd throw `NotSupportedException` or `InvalidOperationException` (`NotImplemented` suggests it'll be implemented at some point)

Reply:
`NotSupportedException` is probably more adequate here.

Reply:
I was actually mistaken; we can already support this. Will implement correctly.

Reply:
Ah, I left it out for simplicity. So let's go with `NotSupportedException` for now.